agent/api/container/container.go

```diff
@@ -66,8 +68,24 @@ const (
 
 	// TargetLogDriver is to show secret target being "LOG_DRIVER", the default will be "CONTAINER"
 	SecretTargetLogDriver = "LOG_DRIVER"
+
+	// Default RestartMaxAttempts OnFailure
+	DefaultRestartMaxAttemptsOnFailure = math.MaxUint32
```

Comment:
is this a sensible default value? 1<<32 - 1 is a big number, practically attempting to restart forever?

Reply:
`RestartMaxAttempts` is only valid for policy: `OnFailure`. If customer doesn't specify `RestartMaxAttempts`, we'll keep restarting it forever unless exit successfully.